pkg/syncer/client.go

```diff
@@ -77,6 +79,11 @@ func (s *RegionSyncer) syncRegion(ctx context.Context, conn *grpc.ClientConn) (C
 
 var regionGuide = core.GenerateRegionGuideFunc(false)
 
+// IsRunningAsClient returns whether the region syncer client is running.
+func (s *RegionSyncer) IsRunningAsClient() bool {
```

Comment:
IsRunning?

Reply:
Since the server and client share a structure, this function only represents the state of the client, and the name IsRunning may be ambiguous

Reply:
Got it. We won't provide `IsRunningAsServer`.